.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -0,0 +1,150 @@
+variables:
+  BENCHMARKS_CI_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/benchmarking-platform:trace-agent
+
+.benchmarks:
```

Comment:
Suggestion: this template job name is generic, so there is a chance that it gets shadowed by mistake by a future change (eg. someone adds another `.benchmarks` template job in another part of the pipeline). Maybe something more specific like `.trace_agent_benchmarks` would work better?